bitnami/kube-prometheus/README.md

```diff
@@ -107,11 +107,14 @@ The following table lists the configurable parameters of the kube-prometheus cha
 | `operator.serviceMonitor.relabelings`                 | Relabel configs                                                                                                     | `[]`                                                                         |
 | `operator.resources`                                  | CPU/Memory resource requests/limits for node                                                                        | `{}`                                                                         |
 | `operator.podAnnotations`                             | Pod annotations                                                                                                     | `{}`                                                                         |
-| `operator.nodeAffinity`                               | Node Affinity (this value is evaluated as a template)                                                               | `{}`                                                                         |
-| `operator.podAntiAffinity`                            | Pod anti-affinity policy                                                                                            | `soft`                                                                       |
-| `operator.podAffinity`                                | Affinity, in addition to antiAffinity (this value is evaluated as a template)                                       | `{}`                                                                         |
```

Comment:
It seems these parameters will disappear and could cause missing sections in the templates while rendering after an upgrade.